lib/omniauth/strategies/clever.rb

```diff
@@ -42,10 +44,23 @@ def callback_phase
         end
       end
 
-      uid{ raw_info['data']['id'] }
+      uid { raw_info.dig(:me, 'data', 'id') }
 
       info do
-        { :user_type => raw_info['type'] }.merge! raw_info['data']
+        personal_info = raw_info[:canonical]
+        first_name = personal_info.dig('data', 'name', 'first')
```

Comment:
Do we use the name at all?

Reply:
Yes, it gets populated into the user's "display name" on Code.org.